### ISSUE TITLE
[NEWS] Zasoby do obsługi zamówień od dziś w wersji public / Orders management resources released in public version

### ISSUE THREAD

**Issue by PrzemyslawLukanowski** [allegro/allegro-api#2220]
Zakończyliśmy okres testowy dla zasobów, dzięki którym jako sprzedawca możesz zarządzać zamówieniami. **Wersję beta będziemy wspierać do 27.09.2019**. W tym okresie możesz systematycznie przechodzić na wersję public. Aby przejść na nową wersję wystarczy zmienić w nagłówkach: 

‘application/vnd.allegro.beta.v1+json’ na **‘application/vnd.allegro.public.v1+json’**. 

**13 stycznia 2020** usuniemy poniższe metody WebAPI, które już dziś oznaczamy jako deprecated:

- [doGetPostBuyData](https://allegro.pl/webapi/documentation.php/show/id,89)
- [doGetPostBuyFormsDataForSellers](https://allegro.pl/webapi/documentation.php/show/id,703)
- [doGetPostBuyFormsIds](https://allegro.pl/webapi/documentation.php/show/id,1362)
- [doGetTransactionsIDs](https://allegro.pl/webapi/documentation.php/show/id,121)
- [doGetSiteJournalDeals](https://allegro.pl/webapi/documentation.php/show/id,742)
- [doGetSiteJournalDealsInfo](https://allegro.pl/webapi/documentation.php/show/id,743)
- [doGetShipmentData](https://allegro.pl/webapi/documentation.php/show/id,624)
- [doAddPackageInfoToPostBuyForm](https://allegro.pl/webapi/documentation.php/show/id,744)
- [doGetDeals](https://allegro.pl/webapi/documentation.php/show/id,462)
- [doGetFilledPostBuyForms](https://allegro.pl/webapi/documentation.php/show/id,722)
- [doGetPaymentMethods](https://allegro.pl/webapi/documentation.php/show/id,726)

Więcej informacji o tym w jak obsługiwać zamówienia przez Allegro REST API znajdziesz w [naszym poradniku](https://developer.allegro.pl/orders/).

-----

We have concluded the test period for resources, which allows you as a seller to manage orders. **We will support the beta version until 27.09.2019.** To switch to the public version, you should change the headers value ‘accept’ from: 

‘application/vnd.allegro.beta.v1+json’ to **‘application/vnd.allegro.public.v1+json’**. 

**On January 13, 2019** we will remove the following methods, which today we have marked as deprecated:

- [doGetPostBuyData](https://allegro.pl/webapi/documentation.php/show/id,89)
- [doGetPostBuyFormsDataForSellers](https://allegro.pl/webapi/documentation.php/show/id,703)
- [doGetPostBuyFormsIds](https://allegro.pl/webapi/documentation.php/show/id,1362)
- [doGetTransactionsIDs](https://allegro.pl/webapi/documentation.php/show/id,121)
- [doGetSiteJournalDeals](https://allegro.pl/webapi/documentation.php/show/id,742)
- [doGetSiteJournalDealsInfo](https://allegro.pl/webapi/documentation.php/show/id,743)
- [doGetShipmentData](https://allegro.pl/webapi/documentation.php/show/id,624)
- [doAddPackageInfoToPostBuyForm](https://allegro.pl/webapi/documentation.php/show/id,744)
- [doGetDeals](https://allegro.pl/webapi/documentation.php/show/id,462)
- [doGetFilledPostBuyForms](https://allegro.pl/webapi/documentation.php/show/id,722)
- [doGetPaymentMethods](https://allegro.pl/webapi/documentation.php/show/id,726)

You will find more information about processing orders in [our tutorial.](https://developer.allegro.pl/en/orders/)

**Comment by barthoos** [comment]
Zmiana jest od juz? Bo znowu, dokumentacja nie nadaza..

**Comment by PrzemyslawLukanowski** [comment]
Tak, zmiana jest od już, dokumentacja jest już zaktualizowana.

**Comment by Maczuga** [comment]
A co z zasobem "order/checkout-forms/{id}/shipments"? W dokumentacji dalej jako beta, zaś jego odpowiednik w WebAPI oznaczony już jako deprecated. Mam nadzieję, że odpowiedni news pojawi się, gdy i ten zasób przejdzie na public.

**Comment by PrzemyslawLukanowski** [comment]
Tak, opublikujemy odpowiedni news w momencie wejścia zasobu na public.

**Comment by TomekDuszynski** [comment]
@PrzemyslawLukanowski zgodnie z informacją od 27.09.2019 metody do obsługi zamówień nie będą działać z nagłówkiem "beta" również na produkcyjnej wersji allegro?

**Comment by PrzemyslawLukanowski** [comment]
Tak, dotyczy to produkcji. Poniżej pełna lista metod:
- [GET /order/events](https://developer.allegro.pl/documentation/#operation/getOrderEventsUsingGET)
- [GET /order/event-stats](https://developer.allegro.pl/documentation/#operation/getOrderEventsStatisticsUsingGET)
- [GET /order/checkout-forms](https://developer.allegro.pl/documentation/#operation/getListOfOrdersUsingGET)
- [GET /order/checkout-forms/{id}](https://developer.allegro.pl/documentation/#operation/getOrdersDetailsUsingGET)
- [GET /order/line-item-id-mappings](https://developer.allegro.pl/documentation/#operation/findMapping)

**Comment by PrzemyslawLukanowski** [comment]
@Maczuga Zasób jest już w wersji public -#2326.
